newsletter-examples/NOAA-CRW-DHW.ipynb

```diff
@@ -0,0 +1,288 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Extraction of a time series of Degree Heating Weeks (DHW) from the NOAA CRW collection\n",
+    "\n",
+    "*This is the notebook used to create the timeseries plot showcased in the DMS Newsletter #2*\n",
+    "\n",
+    "If you want to try this notebook using NECTAR's JupyterHub service just click here (you need to have an academic email registered with AAF): [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ausseabed/dms-newsletter-2/HEAD?filepath=notebooks%2FDHW_timeseries.ipynb)\n",
+    "\n",
+    "What you need to know:\n",
+    "- the location of the Great Barrier Reef you want to assess. It must be a single point (lat, lon)\n",
+    "- the time period you want to assess. It must be a range of years (e.g. 2010-2019)\n",
+    "- the name of the output file you want to create. It must be a string (e.g. 'my_output_file.csv')\n",
+    "- the URI (Uniform Resource Identifier) of the NOAA CRW collection\n",
+    "\n",
+    "In this example we will use the DHW zarr collection managed by the DMS\n",
+    "\n",
+    "First we need to load the libraries we will use:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "outputs": [],
+   "source": [
+    "import xarray as xr\n",
+    "import pandas as pd\n",
```

Comment:
There is no use for `pandas`. It's better to remove `pandas`. If we want to save `zarr ` as `csv`, we could use `xarray ` directly to save it as a csv, something like subzarr.to_dataframe().to_csv("my_output_file.csv").

Reply:
agree